### PR TITLE
ft(S3C-3772): Add support for Utapiv2 client to communicate over TLS

### PR DESCRIPTION
**Pull request by tmacro**
This PR adds some version gating around configuration parsing and the setup of the utapi client.
- Add the `host` config key for v2 clients
- pass through tls configuration for v2 clients

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: lib/utapi/utilities.js [hunk 1]
```diff
@@ -6,7 +6,12 @@ const { UtapiClient, utapiVersion } = require('utapi');
 const logger = require('../utilities/logger');
 const _config = require('../Config').config;
 // setup utapi client
-const utapi = new UtapiClient(_config.utapi);
+const enableClient = utapiVersion === 2 || !!_config.utapi;
+const utapi = new UtapiClient({
+    ..._config.utapi,
+    tls: _config.certFilePaths,
+    enableClient
```
Comment (alexanderchan-scality) [inline lib/utapi/utilities.js]: also looking at https://github.com/scality/utapi/pull/864/files#diff-f9226a8a10ea61b9c4e485d5419770d9001d76c89c884f5ef3d2fd776e11f346R119-R127

`enableClient` doesn't seem to be a param. Should it be `disableClient`?

**Comment by tmacro** [comment]
i'm gonna change this to a draft PR as the changes need a bit more work

**Review by naren-rajendran — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Pull request merged**